events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
you're right :)

Maybe a change on log is a good trade-off.
Best will be to create an internal "cached" list of applications that we managed (sync periodically).